src/PCIMapper.cpp

```diff
@@ -96,7 +96,22 @@ const PCIVendor& PCIMapper::operator[](const std::string& vendor_id) const { ret
 
 // _____________________________________________________________________________________________________________________
 PCIMapper PCI::getMapper() {
-  static std::string path(std::getenv("HOME"));
+
+  const auto getenv = [](const char* name) -> std::string {
+
+#ifdef _MSC_VER
```

Comment:
I don't know Windows and MSVC too well... Does windows even provide an interface to read PCI device and vendor ids? I have implemented this for Linux since the kernel provides these ids rather than vendor/device names...

Reply:
Same issue as the previous. `std::getenv` is marked deprecated on MSVC due to safety issues. It is recommended to use _dupenv_s instead. However, this is not available on gcc.

Reply:
Okay... At the moment the PCIMapper is only used from Linux specific parts of the library and since this class is not part of the high-level API provided by HWinfo, I consider to only compile the source for Linux builds.

Do you think the code will ever be useful for windows builds?